### PR TITLE
Feature/nl/deployment server

### PR DESCRIPTION
**Pull request by NicoleLatifi**
# Turing Index Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Add node and express to package.json in order to create the server necessary for heroku deployment.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing (add-testing to project)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

# Message for reviewer:

Let's see if these changes will allow our app to deploy.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by Leasw144 — APPROVED** [review]
Thanks for driving!

File: package.json [hunk 1]
```diff
@@ -43,5 +47,9 @@
     "ts-node": "~8.3.0",
     "tslint": "~6.1.0",
     "typescript": "~4.0.2"
+  },
+  "engine": {
```
Comment (JoshSevy) [inline package.json:51]: Hopefully per the stack overflow this works. Great work! Thanks for driving through this.

File: server.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//Install express server    
+const express = require('express');
```
Comment (JoshSevy) [inline server.js:2]: Looks great!

**Review by JoshSevy — APPROVED** [review]
Great work! @NicoleLatifi Thank you for adding the express server and dependencies.

**Pull request merged**